modules/tests/test/copy_template_test.go

```diff
@@ -241,27 +196,24 @@ var _ = Describe("Copy template task", func() {
 
 		It("taskrun succeeds and template is updated", func() {
 			config := &testconfigs.CopyTemplateTestConfig{
-				TaskRunTestConfig: testconfigs.TaskRunTestConfig{
-					ServiceAccount: CopyTemplateServiceAccountName,
-				},
+				TaskRunTestConfig: testconfigs.TaskRunTestConfig{},
 				TaskData: testconfigs.CopyTemplateTaskData{
-					SourceTemplateName:      testtemplate.CirrosTemplateName,
-					TargetTemplateName:      NewTemplateName,
-					TargetTemplateNamespace: TestTargetNS,
-					AllowReplace:            "true",
-					Template:                testtemplate.NewCirrosServerTinyTemplate().Build(),
+					SourceTemplateName: testtemplate.CirrosTemplateName,
+					TargetTemplateName: E2ETestsRandomName(NewTemplateName),
```

Comment:
This adds `E2ETestsRandomName`, is that required anywhere else?

Reply:
I updated the code and added the suffix to each target template in copy-template config to prevent flakiness